### PR TITLE
Add Calendly as a Provider

### PR DESCRIPTION
**Pull request by paulchrisluke**
This PR is adds [Calendly](https://calendly.com/) Provider

[Official Calendly API Docs](https://developer.calendly.com/getting-started)
[CalendlyPartners Portal](https://developer.calendly.com/create-a-developer-account)

### CODE REVIEW

File: src/Calendly/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+You should now be able to use the provider like you would regularly use Socialite (assuming you have the facade installed):
+
+```php
+return Socialite::driver('calendly')->redirect();
+```
+
+### Returned User fields
+
+- ``email``
```
Comment (atymic) [inline src/Calendly/README.md:62]: Can you add the additional fields here please?

**Review by atymic — CHANGES_REQUESTED** [review]
One small change

File: src/Calendly/composer.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            "email": "paulchrisluke@gmail.com"
+        }
+    ],
+    "support": {
+        "issues": "https://github.com/socialiteproviders/providers/issues",
+        "source": "https://github.com/socialiteproviders/providers",
+        "docs": "https://socialiteproviders.com/calendly"
+    },
+    "require": {
+        "php": "^8.0",
```
Comment (atymic) [inline src/Calendly/composer.json:24]:
```suggestion
        "php": "^8.1",
```

**Comment by paulchrisluke** [comment]
Thank you @atymic , got that changed 👍🏽

**Review by atymic — APPROVED** [review]

**Pull request merged**